### PR TITLE
Add Linux CI

### PR DESCRIPTION
**Pull request by snnn**

### CODE REVIEW

File: .github/workflows/linux-ci.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: Linux-CPU
+on: [push, pull_request]
```
Comment (edgchen1) [inline .github/workflows/linux-ci.yml:2]: can we specify the main branch for the push/pull_request events?
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestbranchestags

Comment (snnn) [reply .github/workflows/linux-ci.yml:2]: Like
```yaml
on:
  push:
    # Sequence of patterns matched against refs/heads
    branches:    
      # Push events on main branch
      - main
  pull_request:
    # Sequence of patterns matched against refs/heads
    branches:    
      # Push events on main branch
      - main
```

Comment (edgchen1) [reply .github/workflows/linux-ci.yml:2]: yeah, that looks good

Comment (snnn) [reply .github/workflows/linux-ci.yml]: Updated. Please help me check.

File: .github/workflows/linux-ci.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        gcc: [9, 10]
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+      - name: Download onnxruntime binary
+        run: |          
+          aria2c -q -d $RUNNER_TEMP https://github.com/microsoft/onnxruntime/releases/download/v1.8.1/onnxruntime-linux-x64-1.8.1.tgz
```
Comment (gwang-msft) [inline .github/workflows/linux-ci.yml:20]: Put the version number in a variable for easy update later?

**Review by gwang-msft — APPROVED** [review]

**Review by edgchen1 — APPROVED** [review]

**Pull request merged**